### ISSUE TITLE
npm install @nestjs/testing

### ISSUE THREAD

**Issue by stevexm** [nestjs/nest#4836]
## Current behavior

> npm ERR! Unexpected end of JSON input while parsing near '....0","optional":"0.1.4'

## Expected behavior

Expect npm install without errors?

## Possible Solution

Check the @nestjs/testing package.json?
## Environment
<pre><code>
Nest version: 7.1.5

For Tooling issues:
Mac OSX: 10.13.6
npm: 6.14.4
node: 12.16.2

Others:

Thank you!
</code></pre>

**Comment by jmcdo29** [comment]
Works just fine on my machine. 

Linux Mint 19.3 Tricia
Node: 13.14.0
npm: 6.14.4

Usually is a problem with the npm install if I remember. Do other installs still work properly? Can you provide specific reproduction steps?

**Comment by stevexm** [comment]
Hi Jay, thanks for the comment, much appreciated! The two differences between our respective tooling, is me running on osx and node 12.16.2 LTS. 

Following your comment I reinstalled the Nest CLI and then generated a new test project using: 
`nest new testproject-0.2`
After which, all was good and I successfully ran 'Hello World' . 

So, something with my original install. I will close the issue, and am sending apologies for taking up bandwidth with a spurious issue on my end. Thank you again for pointing me in the right direction.
